### PR TITLE
Pine Trees + Updated Taiga

### PR DESCRIPTION
**Pull request by careeoki**
Adds Pine, a new wood type for the Taiga biome.
image
image

### CODE REVIEW

File: assets/cubyz/biomes/taiga.json [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+			"leaves" : "cubyz:pine_needles",
+			"log" : "cubyz:pine_log",
+			"top" : "cubyz:pine_top",
+			"chance" : 0.025,
+			"type" : "pyramid",
+			"height" : 8,
+			"height_variation" : 5
+		}
+	]
+}
```
Comment (IntegratedQuantum) [inline assets/cubyz/biomes/taiga.json:29]: It looks like you have the incorrect line endings again.
Sadly my automated script wasn't detecting this correctly.

Comment (careeoki) [reply assets/cubyz/biomes/taiga.json:29]: Is there a way to run the check again, now that you've fixed it? Or does it do that when the PR is merged?

Comment (IntegratedQuantum) [reply assets/cubyz/biomes/taiga.json:29]: It automatically runs it every time you make a new commit.
But you need the new script in your branch, so you'd have to update your branch to the latest commit from `master`.

**Comment by careeoki** [comment]
Okay, looks like it's passing the check now.

**Pull request merged**